Runtime/CoreSettingsHandler.cs

```diff
@@ -1,71 +1,24 @@
 ﻿using ReadyPlayerMe.Core.Data;
-using UnityEditor;
 using UnityEngine;
 
 namespace ReadyPlayerMe.Core
 {
     public static class CoreSettingsHandler
```

Comment:
I'd rename this to `CoreSettingsLoader` as that's the only thing it does.

Reply:
It still sort of loads it though, except if it's already loaded then it has it to hand so can provide straight away. From the point of view of a piece of consuming code then load would make sense to me. Maybe `CoreSettingsProvider` could work? It just needs to be something that is not as generic as handler (because that means nothing 😅), and instead is something that implies that it's the thing which gives you the core settings (e.g. loader, provider, fetcher etc.)

Reply:
right that makes sense. 
One other thing that also makes me a little hesitant (that I just realized) changing the name of this class in general because  multiple modules/packages use this class (CAC and WebView) so changing this will also require people to update these packages. 
The WebView is not so much an issue because we will be moving the core/shared functionality into core (as per another task).

Reply:
probably if we change the name we should consider it a breaking change. We can also include better handling of breaking changes in this release then?

Reply:
Ah, that's annoying. Is it worth making this change a breaking change which would then need a whole major version bump 🤔 - probably not to be honest 😅 - Maybe we could include renaming this the next time we do have to do a breaking change?

Reply:
Well actually, if we improve our auto updater in the release, we can force it to install the latest of other packages (as required) if they choose to upgrade core. if that makes sense. 
Then we wouldn't need to make it a breaking change. 
However if developers have other scripts somewhere outside the packages that use CoreSettingsHandler then it would still cause issues. 
Perhaps just including in docs or something might be enough

Reply:
I made the change but I just realized that it also makes it difficult to test first time import due to the name change 
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/a2ef0658-eaf5-46b1-b265-daa37a7ffa01)

Reply:
I reverted the class rename. 
If we want to go with it we can do it in a separate PR and probably also include docs or installer updates to try and minimize install errors that this change will cause